### DISCUSSION TITLE
Discord Svelte Brasil

### DISCUSSION THREAD

**Discussion by adamfoerster** [frontendbr/forum#1472] [Dicas]
Olá pessoal.

O framework Svelte tem rapidamente conquitado os corações da galera com sua sintaxe elegante e sua velocidade ultra-rápida.

A comunidade em torno da lib se reune num canal do Discord chamado Svelte. No entanto este canal tem discussões apenas em inglês, e devido ao rápido aumento da popularidade da versão 3 do Svelte, o número diário de mensagens tem sido muito alto.

Pensando em meus colegas lusófonos criei um canal para a comunidade brasileira se ajuntar.

---
Edit 12/jul/2022
Criei um novo que não expira [Svelte Brasil](https://discord.gg/Xf9zzW9).

**Comment by felipefialho** [comment]
Tenho olhado esse framework com bastante atenção 👀

**Comment by didiraja** [comment]
Na Stone Co, o framework usado pra fazer os apps é o Svelte 2. Só ele foi suportado na maquininha que usamos.

Foi feito teste com React, Vue e Preact mas o Svelte venceu com folga pela performance.

**Comment by adamfoerster** [comment]
O convite que mandei acima expirou. Criei um novo que não expira [Svelte Brasil](https://discord.gg/Xf9zzW9).

**Comment by rodrigocode4** [comment]
A sintaxe é realmente incrível, estava olhando a doc e adorei o esquema de tutorial que é na mesma pegada do freeCodeCamp

**Comment by sebastianjnuwu** [comment]
Ainda tem grupo do discord?

**Reply by felquis** [reply]
Não conheço o grupo do Discord, mas no Slack tem uns aficionados por Svelte, link para entrar no Slack está no readme do reposirório frontendbr/forum

**Reply by felquis** [reply]
O invite do Discord ta ali em cima https://github.com/frontendbr/forum/discussions/1472

**Reply by sebastianjnuwu** [reply]
Discord expirou ;-;

**Reply by felquis** [reply]
Olha o comentário https://github.com/frontendbr/forum/discussions/1472#discussioncomment-155074
